### DISCUSSION TITLE
Can I disable the generation of the sitemap?

### DISCUSSION THREAD

**Discussion by leonardehrenfried** [squidfunk/mkdocs-material#4643] [Q&A] [answered]
In particular I want to disable the generation of the gzipped sitemap as on every CI run it generates a commit that appear to regenerate the file.

The commits look like this: https://github.com/opentripplanner/docs/commit/f86a5dc4a3d347084e4acb39fcd7432f449d97e2

**Comment by squidfunk** [comment, marked as answer]
The sitemap is generated by [MkDocs](https://github.com/mkdocs/mkdocs). Please take this question upstream.

**Comment by leonardehrenfried** [comment]
For anyone stumbling across this, the answer is to write a plugin. I did exactly that: https://github.com/leonardehrenfried/mkdocs-no-sitemap-plugin

**Reply by squidfunk** [reply]
Thanks for sharing! Note that [instant loading](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-loading) will not work without `sitemap.xml`.
